build_library/grub.cfg

```diff
@@ -45,7 +45,7 @@ search --no-floppy --set first_boot \
 if [ -n "$first_boot" ]; then
     set first_boot="coreos.first_boot=1"
     # Explicitly request the disk-guid randomization
-    set randomize_disk_guid="coreos.randomize_disk_guid=1"
+    set randomize_disk_guid="coreos.randomize_disk_guid=00000000-0000-0000-0000-000000000001"
```

Comment:
I sortof prefer treating the "randomize disk guid" and "first boot" as entirely independent things